lib/autoload/kuroko2/command/executor.rb

```diff
@@ -19,7 +19,8 @@ def initialize
                    elsif worker_id == (Command::Executor.num_workers - 1)
                      Command::Monitor.new(hostname: @hostname, worker_id: worker_id)
                    else
-                     @worker = Worker.where(hostname: @hostname, worker_id: worker_id, queue: @queue).first_or_create!
+                     @worker = Worker.where(hostname: @hostname, worker_id: worker_id, queue: @queue).first_or_initialize
```

Comment:
`first_or_initialize` doesn't create a new record, is it? Did you try command-executor locally?

Reply:
I did. `first_or_initialize` doesn't create a new record, but `update!` at next line create a new record if not exist.

Reply:
How about `Worker.where(...).first_or_create!(suspendable: true)` ?
I cloudn't get the reason why you split into first_or_initialize + update!.

Reply:
`Worker.where(...).first_or_create!(suspendable: true)` creates a new record with `suspendable: true`, but doesn't update an existing record if the record has `suspendable: false`.

Reply:
OK, but I think creating a new record by update! is confusing